src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -307,9 +307,12 @@
     <value>Cannot compare the value of a token type '{0}' to text.</value>
   </data>
   <data name="FormatDateTime" xml:space="preserve">
-    <value>The JSON value is of unsupported format for a DateTime.</value>
+    <value>The JSON value is of unsupported DateTime format.</value>
```

Comment:
"The JSON value is not in a supported DateTime format."  The current version reads more like "I know what format it is, but I don't support it".  The alternative construction allows for that, or the more likely "this doesn't conform to anything I understand"